atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractTransactionManager.java

```diff
@@ -53,7 +53,7 @@
                 if (!e.canTransactionBeRetried()) {
                     log.warn("[{}] Non-retriable exception while processing transaction.",
                             SafeArg.of("runId", runId),
-                            SafeArg.of("failureCount", failureCount));
+                            e);
```

Comment:
I was just reverting to what was done before.  I could log `e.getMessage()` if you think that's better?